external/Makefile

```diff
@@ -253,6 +253,7 @@ tor/configure:
 
 tor/Makefile: tor/configure
 	cp config.sub tor
+	sed -i '14i #include <errno.h>' tor/src/lib/fs/freespace.c
```

Comment:
Since this is merely a workaround for a bug in tor we should maybe report this and remove it once it has been fixed.

Reply:
Tracking: https://trac.torproject.org/projects/tor/ticket/28286

Reply:
There's a fix on its way (see the ticket) so I think we should wait for the next alpha and remove this?

Reply:
sounds good to me